cmd/insprctl/cli/config_cmd.go

```diff
@@ -46,6 +47,11 @@ func doConfigChange(_ context.Context, args []string) error {
 	}
 
 	// updates
+	if key == cliutils.ServerIpKey() {
+		if !strings.HasPrefix(value, "http://") {
+			value = fmt.Sprintf("http://%s", value)
+		}
+	}
```

Comment:
if i type `https://<my_url>` it would a valid URL but since it doesn't have `http://` is going to add it anyway.

Basically like it is right now:
`https://<my_url>`
will become
`http://https://<my_url>`

A suggestion is to jist check for the `http` preffix, `strings.HasPrefix(value, "http")`

Reply:
```
insprctl config serverip https://34.71.243.160
Success: insprctl config [serverip] changed to 'http://https://34.71.243.160'
```
tried locally to check if the commentary was correct

Reply:
I forgot to change, thanks